app/mutations/application/deleteReviewCommentMutation.ts

```diff
@@ -0,0 +1,54 @@
+import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  updateReviewCommentMutationVariables,
+  updateReviewCommentMutation as updateReviewCommentMutationType
+} from 'updateReviewCommentMutation.graphql';
+import BaseMutation from 'mutations/BaseMutation';
+
+const mutation = graphql`
+  mutation deleteReviewCommentMutation(
+    $input: UpdateReviewCommentInput!
+    $applicationId: ID!
+    $version: String!
+  ) {
+    updateReviewComment(input: $input) {
+      clientMutationId
+      reviewComment {
+        resolved
+        ...ApplicationCommentsByForm_reviewComment
+      }
+      query {
+        application(id: $applicationId) {
+          ...ApplicationDetailsContainer_application
+            @arguments(version: $version)
+        }
+      }
+    }
```

Comment:
remove everything but the query from updateReviewComment(input: $input) {
....
query {...
}

Reply:
Removed